### ISSUE TITLE
CIFAR-10

### ISSUE THREAD

**Issue by junxnone** [junxnone/junxnone.github.io#25]
# [CIFAR Dataset](http://www.cs.toronto.edu/~kriz/cifar.html)

## CIFAR-10 Dataset
The CIFAR-10 dataset consists of 60000 32x32 colour images in 10 classes, with 6000 images per class. There are 50000 training images and 10000 test images. 
![classes](https://github.com/junxnone/junxnone.github.io/blob/master/_img/classes-cifar-10.png?raw=true)

## CIFAR-100 Dataset
This dataset is just like the CIFAR-10, except it has 100 classes containing 600 images each. There are 500 training images and 100 testing images per class. The 100 classes in the CIFAR-100 are grouped into 20 superclasses. Each image comes with a "fine" label (the class to which it belongs) and a "coarse" label (the superclass to which it belongs).

Superclass | Classes
-- | --
aquatic mammals | beaver, dolphin, otter, seal, whale
fish | aquarium fish, flatfish, ray, shark, trout
flowers | orchids, poppies, roses, sunflowers, tulips
food containers | bottles, bowls, cans, cups, plates
fruit and vegetables | apples, mushrooms, oranges, pears, sweet peppers
household electrical devices | clock, computer keyboard, lamp, telephone, television
household furniture | bed, chair, couch, table, wardrobe
insects | bee, beetle, butterfly, caterpillar, cockroach
large carnivores | bear, leopard, lion, tiger, wolf
large man-made outdoor things | bridge, castle, house, road, skyscraper
large natural outdoor scenes | cloud, forest, mountain, plain, sea
large omnivores and herbivores | camel, cattle, chimpanzee, elephant, kangaroo
medium-sized mammals | fox, porcupine, possum, raccoon, skunk
non-insect invertebrates | crab, lobster, snail, spider, worm
people | baby, boy, girl, man, woman
reptiles | crocodile, dinosaur, lizard, snake, turtle
small mammals | hamster, mouse, rabbit, shrew, squirrel
trees | maple, oak, palm, pine, willow
vehicles 1 | bicycle, bus, motorcycle, pickup truck, train
vehicles 2 | lawn-mower, rocket, streetcar, tank, tractor

# Reference

**Comment by junxnone** [comment]
# Test Result

## Local Host
[cifar10_cnn.py](https://github.com/keras-team/keras/blob/master/examples/cifar10_cnn.py)
```
Epoch 100/100
1563/1563 [==============================] - 91s 58ms/step - loss: 0.8168 - acc: 0.7310 - val_loss: 0.7738 - val_acc: 0.7540
Saved trained model at /home/junx/tf/keras/examples/saved_models/keras_cifar10_trained_model.h5 
10000/10000 [==============================] - 5s 495us/step
Test loss: 0.7737732452392578
Test accuracy: 0.754
```

**Comment by junxnone** [comment]
[cifar10_cnn_tfaugment2d.py](https://github.com/keras-team/keras/blob/master/examples/cifar10_cnn_tfaugment2d.py)

# Test Result
## Local Host
Epoch 100/100
50000/50000 [==============================] - 91s 2ms/step - loss: 0.7431 - acc: 0.7538 - val_loss: 0.7650 - val_acc: 0.7540
Saved trained model at /tmp/saved_models/keras_cifar10_trained_model.h5 
10000/10000 [==============================] - 5s 533us/step
Test loss: 0.7611267491340638
Test accuracy: 0.7566
